docs/.vitepress/theme/custom.css

```
@@ -8,9 +8,22 @@
     --vp-c-bg-soft: oklch(94% 0.018 218);
     --vp-c-border: oklch(82% 0.025 218);
     --vp-c-divider: oklch(86% 0.022 218);
-    --vp-font-family-base: "Avenir Next", Avenir, "Segoe UI", sans-serif;
+    --vp-font-family-base: "Geist Variable", ui-sans-serif, sans-serif;
     --vp-font-family-mono:
         "SFMono-Regular", Consolas, "Liberation Mono", monospace;
```

Stylelint (17.14.0)

[error] Expected "Consolas" to be "consolas" (value-keyword-case)

(value-keyword-case)